accepted/0022-git-buildpack.md

```diff
@@ -0,0 +1,61 @@
+# Git Support
+
+## Proposal
+
+Introduce an (optional) Git buildpack into each language buildpack family. The buildpack
+will be charged with providing `git` metadata as build-time and run-time environment
+variables.
+
```

Comment:
It would be really great if this buildpack could also contribute the following labels, [defined in the OCI spec](https://github.com/opencontainers/image-spec/blob/master/annotations.md#pre-defined-annotation-keys)
* org.opencontainers.image.source (public repos only?)
* org.opencontainers.image.revision
* org.opencontainers.image.version (maybe? if the commit is tagged?)

Currently the https://github.com/paketo-buildpacks/image-labels buildpacks allows users to set these explicitly but it would be a better experience for users if it just worked in cases where a `.git` directory was present.

We don't need to make this a condition of accepting the RFC but just wanted to suggest the idea while we are discussing this buildpack :)